docarray/index/backends/hnswlib.py

```diff
@@ -332,7 +340,19 @@ def _filter(
         limit: int,
     ) -> DocList:
         rows = self._execute_filter(filter_query=filter_query, limit=limit)
-        return DocList[self.out_schema](self._doc_from_bytes(blob) for _, blob in rows)  # type: ignore[name-defined]
+        hashed_ids = [doc_id for doc_id, _ in rows]
+        embeddings: OrderedDict[str, list] = OrderedDict()
```

Comment:
Why specifically OrderedDict? I think normal dict in python will already be ordered by default

Reply:
I would still like to be specific about the requirement for order